paas_app_charmer/_gunicorn/charm.py

```diff
@@ -49,12 +49,14 @@ def __init__(self, framework: ops.Framework, wsgi_framework: str) -> None:
             wsgi_framework: WSGI framework name.
         """
         super().__init__(framework)
+        self._wsgi_framework = wsgi_framework
+
         self._secret_storage = GunicornSecretStorage(
             charm=self, key=f"{wsgi_framework}_secret_key"
         )
         self._database_requirers = make_database_requirers(self, self.app.name)
         try:
-            wsgi_config = self.get_wsgi_config()
+            _ = self.get_wsgi_config()
```

Comment:
I think `_ = ` can be removed altogether

Reply:
Removed in the decorator. Thanks!